src/Component.php

```diff
@@ -121,12 +129,18 @@ protected function mergeContext(array $base, array $override): array {
    *   The parsed component definition.
    */
   protected function loadDefinition(string $pathname) {
+    $component_data = [];
     $definition_pathname = $this->getDefinitionFilePath($pathname);
-    return Yaml::parse(file_get_contents($definition_pathname));
+    if (file_exists($definition_pathname)) {
```

Comment:
There was an issue when the config file did not exist an error was thrown. So I added an additional `file_exist` check.

Reply:
What's the behavior if the config file doesn't exist?  Is the error reported?

Reply:
Is it an error to begin with?

Reply:
No it's not reported. Also I don't see necessity as some components don't have a config file by purpose.